Source/Fuse.Controls.Video/Video.ScriptClass.uno

```diff
@@ -17,6 +17,20 @@ namespace Fuse.Controls
 				new ScriptMethod<Video>("stop", stop, ExecutionThread.MainThread));
 		}
 
+		object _positionMutex = new object();
+		object _durationMutex = new object();
+
+		// private fields used by ScriptClass for retrieving the position from JavaScript
+		double _outPosition;
+		double _outDuration;
+		void UpdateScriptClass(double position, double duration)
+		{
+			lock (_positionMutex)
+				_outPosition = position;
+			lock (_durationMutex)
+				_outDuration = duration;
```

Comment:
I don't think we need two mutexes for these, one should be enough... It's not a huge contention, two assigns vs one.

Reply:
aaight 👍